AFNI clusterwise p<0.05, cluster connectivity NN2 and NN3

RuthPauli opened the pull request:
This PR creates:
- AFNI example FWE p<0.05
- AFNI example with cluster-connectivity on edges and faces (NN=2 in AFNI)
- AFNI example with cluster-connectivity on edges, faces and corners (NN=3 in AFNI)

All of those were performed on ds011 from [BIDS-compliant datasets](https://drive.google.com/folderview?id=0B2JWN60ZLkgkMGlUY3B4MXZIZW8&usp=sharing).

RuthPauli committed:
AFNI FWER p<0.05 cluster-wise

RuthPauli committed:
Merge branch 'master' of https://github.com/incf-nidash/nidmresults-examples

RuthPauli committed:
AFNI NN2

RuthPauli committed:
AFNI NN3

cmaumet commented:
@RuthPauli: I have reviewed NN2 and NN3 and they look good!

I just have one minor comment for FWER p<0.05 cluster-wise. I think you used a minimum cluster size of `13` (cf. [batch.sh](https://github.com/incf-nidash/nidmresults-examples/blob/afni_ex/afni_clusterwise_fwe/batch.sh#L1)), but looking into the [ClustSim.NN1_2sided.1D table](https://github.com/incf-nidash/nidmresults-examples/blob/d7e9bf703b34911ea3372c07be828ed56fe9a411/afni_clusterwise_fwe/sub_001.results/ClustSim.NN1_2sided.1D), the value corresponding to p<0.05 FWE-corrected with a cluster-forming threshold of p<0.001 is `11.6`, so should the minimum cluster size be set to `12` instead?

Could you also create a README file for each of these?

Once those minor corrections are done, I think that this will be ready for merge!

RuthPauli commented:
Hi Camille,

You are right. 13 was from the 1-sided test. I will correct that now.

I am making the README files, but it is taking a little while because I can't figure out exactly what one of the conditions was.

Ruth

RuthPauli committed:
README files, file paths corrected, FWER cluster size corrected

These changes:
-Add README files to each AFNI folder
-Correct file paths in scripts
-Correct cluster size for AFNI FWER test

cmaumet commented:
@RuthPauli: this looks good! Let's merge!
